src/main/java/com/typee/model/ModelManager.java

```diff
@@ -85,36 +85,36 @@ public void setAddressBookFilePath(Path addressBookFilePath) {
 
     @Override
     public void setAddressBook(ReadOnlyAddressBook addressBook) {
-        this.addressBook.resetData(addressBook);
+        this.statedAppointmentList.resetData(addressBook);
```

Comment:
in our next update, we should rename all adressbook related attributes and methods to typee. But as of now, i think it is fine to keep it this way as long it is working. good job.

Reply:
Agreed. I'll work on refactoring at a later time.